### PR TITLE
adding util method to remove accents from input msg

### PR DESCRIPTION
**Pull request by monoguerin**
Fixing #10 bug

### CODE REVIEW

File: src/util/removeAccents.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+const characterMap = {
```
Comment (andreszorro) [inline src/util/removeAccents.js]: I think we can leverage this library instead, to handle more cases, like `ü` https://www.npmjs.com/package/diacritics

Comment (monoguerin) [inline src/util/removeAccents.js]: I thought that but not sure is that really necessary, I mean in what cases are you going to use a `ç` or a `ü` at least for now...

Comment (andreszorro) [inline src/util/removeAccents.js]: This library has a small footprint, and we can use it in the future on for more conversational approaches.

File: src/util/removeAccents.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	'á': 'a',
+	'é': 'e',
+	'í': 'i',
+	'ó': 'o',
+	'ú': 'ú'
```
Comment (nmunozsi) [inline src/util/removeAccents.js]: Change 'ú' with 'u'

Comment (monoguerin) [inline src/util/removeAccents.js]: 😞 sorry I missed that,  @andreszorro is ok to still use this "util" method or should we move to the module you suggest?

Comment (andreszorro) [inline src/util/removeAccents.js]: I think the library usage will prevent us for tiny mistakes like this one in the future. :)

Comment (monoguerin) [inline src/util/removeAccents.js]: will move then to that library

**Pull request merged**